package.json

```diff
@@ -20,6 +20,7 @@
     "graphql": "^14.4.2",
     "identicon.js": "^2.3.3",
     "insane": "^2.6.2",
+    "lodash": "^4.17.15",
```

Comment:
I think we're only using throttle. can we just use 
`lodash.throttle@4.1.1`

1kb vs 24kb